CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * **IMPROVEMENT** (by @imaNNeoFighT): Use Container to draw axis-based charts border.
 * **FEATURE** (by @FlorianArnould) Add the ability to select the RadarChart shape (circle or polygon), #1047.
 * **BUGFIX** (by @imaNNeoFighT): Fix LineChart titles problem with single FlSpot, #1053.
+* **BREAKING** (by @FlorianArnould) Add the ability to rotate the RadarChar titles, #883. This breaks [RadarChartData.getTitle](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/radar_chart.md#RadarChartData) which have a new parameter `angle` and now returns a [RadarChartTitle](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/radar_chart.md#RadarChartTitle) instead of a simple `string`. (Read our [Migration Guide](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/migration_guides/MIGRATION_NEW_VERSION.md) to learn more about it)
```

Comment:
It would be great if we have two separate lines. One for our new **FEATURE**. And another line below that for **BREAKING**. Instead of combining both in a single line.

Reply:
I split it into 2 different lines